html/index.html

```diff
@@ -45,6 +45,7 @@
 
 <script type="text/javascript" src="js/lib/jquery-1.9.1.min.js?v={{version}}"></script>
 <script type="text/javascript" src="js/lib/jquery-ui-1.10.1.custom.min.js?v={{version}}"></script>
+<script> if (! ("ontouchend" in document)){document.ontouchend=''}</script>
```

Comment:
So it turns out `jquery.ui.touch-punch.min.js` is broken as it checks for this and bails out if not set. No idea why. Perhaps an old touch events thing or only certain browsers? Who knows.